src/scicookie/cookiecutter.json

```
@@ -59,6 +59,7 @@
   "use_ruff": "no",
   "use_shellcheck": "no",
   "use_vulture": "no",
+  "use_git": "yes",
```

```suggestion
  "use_git": "no",
```

it should be no by default in the cookiecutter.json